zilliqa/tests/it/consensus.rs

```diff
@@ -24,7 +24,7 @@ async fn network_can_die_restart(mut network: Network) {
                 let index = n.random_index();
                 n.get_node(index).get_finalized_height() >= start_block
             },
-            50,
+            100,
```

Comment:
Are all the changes to the tests still necessary?
If they make things more reliable then :+1: